code/modules/clothing/neck/apron.dm

```diff
@@ -10,40 +10,44 @@
 
 /obj/item/clothing/neck/apron/bartender
 	name = "bartenders apron"
-	desc = "A fancy purple apron for a stylish bartender. Can hold some bottles, a few kitchen trinkets and maybe a small discrete revolver...."
+	desc = "A fancy purple apron for a stylish bartender. Has stylish loops for bottles and snacks, sheathes for kitchen trinkets, and a discrete little holster under a flap. Service with a smile!"
```

Comment:
```suggestion
	desc = "A fancy purple apron for a stylish bartender. Has stylish loops for bottles and snacks, sheaths for kitchen trinkets, and a discrete little holster under a flap. Service with a smile!"
```
Surprisingly, sheaths is the plural of sheath.

Reply:
Weird, would think I'd know that by now lol